pyproject.toml

```diff
@@ -40,6 +40,9 @@ Repository = "https://github.com/insanum/gcalcli"
 [project.optional-dependencies]
 vobject = ["vobject"]
 
+[tool.setuptools]
+packages = ["gcalcli"]
```

Comment:
FYI this is needed because package paths are ambiguous after adding a top-level stubs/ dir.